libzecale/circuits/verification_key_hash_gadget.hpp

```diff
@@ -16,65 +18,19 @@ namespace libzecale
 /// on a hash gadget `hashT` which operates directly on arrays of scalars,
 /// producing digests which are also scalars.
 template<typename wppT, typename nverifierT>
-class verification_key_scalar_hash_gadget
-    : public libsnark::gadget<libff::Fr<wppT>>
+class verification_key_hash_gadget : public libsnark::gadget<libff::Fr<wppT>>
```

Comment:
Note to self: `verification_key_scalar_hash_gadget` is renamed `verification_key_hash_gadget`, so the default VK hashing behavior is using an algebraic hash like MiMC defined over the relevant prime field. We remove support for the "standard" VK hasher here.